foreign_cc/private/cmake_script.bzl

```diff
@@ -130,6 +131,7 @@ def create_cmake_script(
         "CMAKE_BUILD_TYPE": build_type,
         "CMAKE_INSTALL_PREFIX": install_prefix,
         "CMAKE_PREFIX_PATH": merged_prefix_path,
+        "PKG_CONFIG_ARGN": "--define-variable=EXT_BUILD_DEPS=$$EXT_BUILD_DEPS$$",
```

Comment:
I've found that building https://github.com/capstone-engine/capstone produces a capstone.pc file with `${EXT_BUILD_ROOT}` left in. So should this also define that variable?